component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java

```diff
@@ -810,13 +823,19 @@ public void fileSinkTestGroup() throws InterruptedException {
                     count.incrementAndGet();
                 }
             }
-            AssertJUnit.assertEquals(4, count.intValue());
+            AssertJUnit.assertEquals(2, count.intValue());
         } else {
             AssertJUnit.fail(sinkUri + " is not a directory.");
         }
 
         Thread.sleep(1000);
         siddhiAppRuntime.shutdown();
+        File sinkRoot = new File(sinkUri);
+        try {
+            FileUtils.deleteDirectory(sinkRoot);
+        } catch (IOException e) {
+            throw new TestException("Failed to delete files in due to " + e.getMessage(), e);
```

Comment:
log the sinkUri too

Reply:
Fixed in cdb7b86019f370bcb90c545043b2ad205e8e98df